Server/src/main/java/org/gluu/oxauth/register/ws/rs/RegisterRestWebServiceImpl.java

```diff
@@ -143,7 +143,7 @@ public Response requestRegister(String requestParams, HttpServletRequest httpReq
     }
 
     private Response registerClientImpl(String requestParams, HttpServletRequest httpRequest, SecurityContext securityContext) {
-        Response.ResponseBuilder builder = Response.ok();
+        Response.ResponseBuilder builder = Response.status(201);
```

Comment:
We need backward compatibility support.
```
Response.ResponseBuilder builder = Response.status(CREATED);
if (appConfiguration.getReturn200OnClientRegistration()) {
    builder = Response.ok();
}
```

Reply:
I think this is excellent, I can work on it.

Reply:
I think this is excellent, I can work on it.